src/main/java/org/apache/xml/security/parser/XMLParser.java

```diff
@@ -27,15 +27,6 @@
  */
 public interface XMLParser {
 
-    /**
-     * Parses a document from the input stream.
-     * Caller is responsible for closing the stream.
-     *
-     * @param inputStream
-     * @param disallowDocTypeDeclarations
-     * @return {@link Document}
-     * @throws XMLParserException
-     */
```

Comment:
Why removing it?

Reply:
Mistake - I made a separate PR for master and immediately rebased and reverted unnecessary changes here. Unfortunately I forgot that it is a sequence of commits so the patch was applied by git in the next rebase without conflicts.
Thanks for noticing that!